### PR TITLE
Add ChatGPT config in tag.

### PR DESCRIPTION
**Pull request by nodeloc**
I add a ChatGPT config in tag to enable which tag can use ChatGPT.

### CODE REVIEW

File: src/Listener/PostChatGPTAnswer.php [hunk 1]
```diff
@@ -52,7 +65,7 @@ public function handle(Started $event): void
 
         $firstPost = $discussion->firstPost;
 
-        $content = $this->openAI->completions($firstPost->content);
+        $content = $this->openAI->completions($firstPost->title.$firstPost->content);
```
Comment (datlechin) [inline src/Listener/PostChatGPTAnswer.php:68]: I think it should have a space between title and content 
```suggestion
        $content = $this->openAI->completions(“{$firstPost->title} {$firstPost->content}”);
```

File: js/src/admin/index.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import app from 'flarum/admin/app';
+import addChatgptToTagsModal from './addChatgptToTagsModal';
```
Comment (datlechin) [inline js/src/admin/index.tsx:2]: Where is the file `addChatgptToTagsModal`

**Comment by nodeloc** [comment]
Sorry I push a wrong branch. I will push my main branch later.

**Comment by datlechin** [comment]
@nodeloc Ok, I will closed this PR

**Pull request closed without merging**